challenges/0-hello-world/php/sorenhe4rt/hello.php

```diff
@@ -0,0 +1,2 @@
+<?php
+echo'Hello World';
```

Comment:
Salve primo! 
Seu código está 100% funcional! Porém, gostaria muito de ver a aplicação da [PSR-12](https://www.php-fig.org/psr/psr-12/) dentro do seu código.

Se trata de seguir algumas regras para que o código fique o mais claro possível para quem está lendo e no seu caso foi apenas um erro bobo. Fica a sugestão para aprovarmos seu PR!

```suggestion
echo 'Hello World';
```